lib/paperclip/storage/s3.rb

```diff
@@ -113,18 +113,23 @@ module Storage
 
     module S3
       def self.extended base
-        begin
-          require 'aws-sdk'
-          const_set('AWS_CLASS', defined?(::Aws) ? ::Aws : ::AWS)
-          const_set('AWS_BASE_ERROR',
-            defined?(::Aws) ? Aws::Errors::ServiceError : AWS::Errors::Base)
-          const_set('DEFAULT_PERMISSION',
-            defined?(::AWS) ? :public_read : :'public-read')
-
-        rescue LoadError => e
-          e.message << " (You may need to install the aws-sdk gem)"
-          raise e
-        end unless defined?(AWS_CLASS)
+        unless defined?(AWS_CLASS)
+          begin
+            require 'aws-sdk'
+            const_set('AWS_CLASS', defined?(::Aws) ? ::Aws : ::AWS)
+            const_set('AWS_BASE_ERROR',
+              defined?(::Aws) ? Aws::Errors::ServiceError : AWS::Errors::Base)
+            const_set('DEFAULT_PERMISSION',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.